README.md

```diff
@@ -10,7 +10,7 @@
 |---------|---------|----|-----------|
 |type     |True |it can be `deb` or `rpm` |type=rpm|
 |app      |True |it can be `apisix` or `dashboard` |app=apisix|
-|code_tag   |True |the code tag of the app which you want to package, it can not be `master`|code_tag=2.1|
+|code_tag   |True |the code tag of the app which you want to package|code_tag=2.1|
```

Comment:
Not only for tag, but could also work for branch. Maybe we need to change the argument name

Reply:
If it is a branch, it is different. For example, the branch of APISIX is `v2.3` but the tag is 2.3. Their processing in the dockerfile is different.
```
&& git checkout ${apisix_tag} \
```
should be
```
&& git checkout "v"${apisix_branch} \
```

Reply:
If it is a branch, it is different. For example, the branch of APISIX is `v2.3` and the tag is `2.3`. Their processing in the dockerfile is different.
```
git checkout ${apisix_tag}
```
should be
```
git checkout "v"${apisix_branch}
```
so, If you want to do it, you may need a new parameter.

Reply:
agree @Yiyiyimu's way. do you have any advice?

Reply:
> I think we only need to rename the code_tag, like checkout?

agree. what do you think @idbeta

Reply:
BTW, I just find that git checkout `2.3` and `v2.3` would give us a different result is weird to me. 
The branch is not only not synced with tag but also should be replaced by tag since it's stale. I don't think we need to keep branches. But seems people on mailing list does not agree on this issue